runtime/src/bank.rs

```diff
@@ -2524,6 +2534,117 @@ impl Bank {
         }
     }
 
+    #[allow(dead_code)]
+    // Calculate rewards from previous epoch and distribute vote rewards
+    fn calculate_rewards_and_distribute_vote_rewards(
```

Comment:
this _is_ the caller that does both calculate and store votes. That makes the caller of this, coming soon, simpler than it would be. This function is just one level higher in the chain. We are almost to the top.